src/resources/js/ian-client.js

```diff
@@ -1,14 +1,30 @@
 ( function ( Ian ) {
 	window.addEventListener( 'load', function ( event ) {
-		Ian.icon = document.querySelector( '[data-tec-ian-trigger="iconIan"]' );
-		Ian.sidebar = document.querySelector( '[data-tec-ian-trigger="sideIan"]' );
-		Ian.notifications = document.querySelector( '[data-tec-ian-trigger="notifications"]' );
-		Ian.readAll = document.querySelector( '[data-tec-ian-trigger="readAllIan"]' );
-		Ian.optin = document.querySelector( '[data-tec-ian-trigger="optinIan"]' );
-		Ian.close = document.querySelector( '[data-tec-ian-trigger="closeIan"]' );
-		Ian.empty = document.querySelector( '[data-tec-ian-trigger="emptyIan"]' );
-		Ian.loader = document.querySelector( '[data-tec-ian-trigger="loaderIan"]' );
-		Ian.consent = Ian.notifications.dataset.consent;
+		// Ian.icon will be queried later, as it is dynamically created.
+		// All others should be present in the DOM when 'load' fires.
+		Ian.sidebar = document.querySelector(
```

Comment:
Why all these line splits? I don't think they are long enough to justify splitting to a new line.